prepare-commit-msg

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# An example hook script to prepare the commit log message.
+# Called by "git commit" with the name of the file that has the
+# commit message, followed by the description of the commit
+# message's source.  The hook's purpose is to edit the commit
+# message file.  If the hook fails with a non-zero status,
+# the commit is aborted.
+#
+# To enable this hook, rename this file to "prepare-commit-msg".
+
+# This hook includes three examples. The first one removes the
+# "# Please enter the commit message..." help message.
+#
+# The second includes the output of "git diff --name-status -r"
+# into the message, just before the "git status" output.  It is
+# commented because it doesn't cope with --amend or with squashed
+# commits.
+#
+# The third example adds a Signed-off-by line to the message, that can
+# still be edited.  This is rarely a good idea.
+
+COMMIT_MSG_FILE=$1
+COMMIT_SOURCE=$2
+SHA1=$3
+
+sed -i "" "s/.*Instance generated from template:.*/.. Instance generated from template: $(git rev-parse HEAD)/" README.rst
```

Comment:
this is just an example of a hook that could be used to embed the commit hash

Reply:
ok, we need the commit has of the cookiecutter template in some machine-readable form. Either as a file ``.invenio`` or e.g. in ``version.py`` as you suggested.